test/p2p/peer.sh

```diff
@@ -21,7 +21,7 @@ if [[ "$ID" == "x" ]]; then # Set "x" to "1" to print to console.
 		--ip=$(test/p2p/ip.sh "$ID") \
 		--name "local_testnet_$ID" \
 		--entrypoint tendermint \
-		-e TMHOME="/go/src/github.com/tendermint/tendermint/test/p2p/data/mach$ID/core" \
+		-e TMHOME="/go/src/github.com/tendermint/tendermint/test/p2p/data/mach$((ID-1))" \
```

Comment:
write a comment about where's mach data is coming from (because it's not in the repo anymore)